client/src/components/Ticket.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import {Card, CardText, CardHeader, CardActions} from 'material-ui/Card';
+import RaisedButton from 'material-ui/RaisedButton';
+
+class Ticket extends Component {
+	
+
+	render()
+	{
+
+		return(
+			<Card>
+				<CardHeader title = "Username"
+					subtitle="When request was submitted"/>
+				<CardText style = "color:red;">
+					Issue <br/>
+					Location <br/>
+					Contact Number<br/>
+				</CardText>
+				<CardActions>
+
+					<RaisedButton label="REOPEN TICKET"
+						backgroundColor="#F48FB1"/>
+					<br />
```

Comment:
Hm maybe remove these break tags and we can add css styling later to create some margin between the buttons.